opengrok-indexer/src/main/java/org/opengrok/indexer/index/Indexer.java

```diff
@@ -162,6 +163,9 @@ public static void main(String[] argv) {
                 pauseToAwaitProfiler();
             }
 
+            disabledRepositories.addAll(cfg.getDisabledRepositories());
```

Comment:
perhaps logprint (`FINEST` ?) all disabled repositories at the end of `parseOptions()` for debugging purposes ?

Reply:
OK added